api/File.json

```diff
@@ -148,7 +148,7 @@
               "version_added": "16"
             },
             "opera_android": {
-              "version_added": false
+              "version_added": "16"
```

Comment:
Can you re-mirror all of the Opera data? It's inconsistent, Chrome/Android 13/18 versions don't map to the same Opera versions on different features.